data/model/oci/tag.py

```diff
@@ -183,6 +184,16 @@ def list_repository_tag_history(
         .limit(page_size + 1)
         .offset(page_size * (page - 1))
     )
+    try:
+        if filter_tag_name is not None:
+            operation, value = filter_tag_name.split(':')
```

Comment:
consider using `split(':', 1)` so you only have 2 possible entries. if someone passes a `:` in the query then you'll have 3 entries in the split which will cause an exception here

Reply:
Nice one! I've updated to reflect the same.